argilla-server/src/argilla_server/api/schemas/v1/fields.py

```diff
@@ -88,11 +88,30 @@ class ChatFieldSettingsUpdate(BaseModel):
     use_markdown: bool
 
 
+class CustomFieldSettings(BaseModel):
+    type: Literal[FieldType.custom]
+    template: str
+    advanced_mode: bool
+
+
+class CustomFieldSettingsCreate(BaseModel):
+    type: Literal[FieldType.custom]
+    template: str
```

Comment:
We don't need to add any length validation to the template? I mean an empty string is a valid template here.